modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/status-content/reviewed-rejected-content/index.tsx

```diff
@@ -49,6 +60,32 @@ const UnderwritingContent = ({
 		redirectTo('Applications');
 	};
 
+	useEffect(() => {
+		getUserNotification().then((response) => {
+			const notifications = response?.data?.items;
+
+			const message = notifications.filter(
+				(notification: ConstantListType) => {
+					if (notification.message.includes(`${id}`)) {
+						return notification.message;
+					}
+				}
+			);
+
+			const comment = message[0]?.message?.split(': ')[1];
+
+			setUnderwriterComment(comment);
+		});
+	}, [id]);
+
+	const handleShowComment = () => {
```

Comment:
I think you just need one validation here. 
```suggestion
	const handleShowComment = () => { underwriterComment ? underwriterComment : "This is uninsurable"}
```

Reply:
In this case, this validation is necessary.